README.md

```diff
@@ -20,10 +20,68 @@ If you are looking to use ssb-server to run a pub, consider using [ssb-minimal-p
 
 ## Install
 
-To add `ssb-server` to your available CLI commands, install it using the `-g` global flag:
+How to Install `ssb-server` and create a working pub 
+
+1. `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash`
+
+2. `npm install -g node-gyp`
+
+3. `apt-get install autotools-dev automake`
+
+4. `nvm install 10`
+
+5. `nvm alias default 10`
+
+6. Then to add `ssb-server` to your available CLI commands, install it using the `-g` global flag:
 ```
 npm install -g ssb-server
 ```
+7. `nano ~/run-server.sh` and input:
+
+```
+#!/bin/bash
+while true; do
+  ssb-server start
+  sleep 3
+done
+```
+
+Be sure to start the pub server from this script, as this script will run the pub server and restart it even if it crashes.      
+
+8. `mkdir ~/.ssb/`
+
+9. `nano ~/.ssb/config` and input:
+
+```
+{
+  "connections": {
+    "incoming": {
+      "net": [
+        { "scope": "public", "host": "0.0.0.0", "external": ["Your Host Name or Public IP"], "transform": "shs", "port": 8008 }
+      ]
+    },
+    "outgoing": {
+      "net": [{ "transform": "shs" }]
+    }
+  }
+}
+```
+
+10. Now run `sh ~/run-server.sh` in a detachable  session (e.g. screens)
+
+11. Detach the session and run `ssb-server whoami` to check to see if the server is working.
+
+12. Now is the time to think of a really cool name for your new pub server.  Once you have it run:
+
+`ssb-server publish --type about --about {pub-id (this is the output from ssb-server whoami)} --name {Your pubs awesome name}`
+
+12. Now it's time to create those invites! 
+Just run `ssb-server invite.create 1` and send those codes to your friends.
+
+Congratulations!  You are now ready to scuttlebutt with your friends! 
+
+P.S. if you get the error that your invite code is not valid, when using the invite code on a client trying to connect to your server, try opening port 8008 on your router, it’s what worked for me.
```

Comment:
I am trying to imagine why it could be the case that this worked... it seems more like a glitch than the fix to me. 
The port 8008 in the config file shouldn't have any implications for client nodes trying to connect I don't think.
I wonder if we could ask @mixmix about this.

Reply:
How certain are you that this was the fix for you? 
Because that's not something that's easy for a lot of people, and wouldn't want anyone trying to go through that step if its a false trail

Reply:
Specifically what happened is when I would try the invite code the server generated on both manyverse and patchwork I would get an error that the code was invalid, but once I opened port 8008 on my router they both worked on the first try. 

So I am fairly sure that opening port 8008 on my router is what fixed the problem, I will close the port and see if the clients disconnect just to be sure.

Reply:
Gotcha! I learned something, and I'm glad you checked it out. 

By the way, when you say 'opened the port' on your router, do you mean port forwarding, or what kind of thing?

Reply:
Yeah, its just port forwarding port 8008.